style.css

```diff
@@ -193,16 +280,230 @@ h3 {
   background-color: #1a2236;
 }
 
-/* Aboutme Section css */
+/* Works Desktop Section CSS */
+
+@media (min-width: 768px) {
+  .works {
+    grid-template-columns: 26.8% 19.8% 16.8% 19.8% 16.8%;
+    grid-template-rows: 32.2% 4.8% 10.6% 10.6% 10.6% 15.6% 15.6%;
+    grid-auto-flow: row;
+    gap: 0 0;
+    height: 1660px;
+  }
+
+  .works img {
+    display: none;
+  }
+
+  .buttonproject img {
+    display: block;
+  }
+
+  h3 {
+    font-weight: 600;
+    font-size: 30px;
+  }
+
+  .html {
+    width: 132px;
+  }
+
+  .ruby {
+    width: 169px;
+  }
+
+  .js {
+    width: 144px;
+  }
+
+  .html,
+  .ruby,
+  .js {
+    height: 40px;
+    font-size: 20px;
+    line-height: normal;
+  }
+
+  .buttonproject {
+    height: 51px;
+    width: 212px;
+  }
+
+  .projecttitle {
+    display: flex;
+    align-items: center;
+    background-image: url(img/imagedots.svg);
+    background-position: 100% 80%;
+    background-repeat: no-repeat;
+    background-size: 130px;
+    transform: rotate(-90deg);
+    font-size: 110px;
+    margin: 0;
+    grid-area: 1 / 1 / 2 / 2;
+  }
+
+  .projectdescription.a {
+    grid-area: 1 / 2 / 2 /6;
+    display: flex;
+    justify-content: flex-end;
+    align-items: flex-start;
+    background-image: url(img/project1.png);
+    background-size: cover;
+    background-repeat: no-repeat;
+  }
+
+  .projectinfo1 {
+    display: flex;
+    flex-direction: column;
+    background-color: #1a2236;
+    width: 100%;
+    height: 60%;
+    align-items: flex-start;
+    justify-content: space-evenly;
+    padding-left: 6%;
+  }
+
+  .projectinfo1 h3 {
+    font-size: 40px;
+  }
+
+  .projectdescription.b {
+    grid-area: 3 / 1 / 5 / 2;
+    display: flex;
+    justify-content: flex-end;
+    background-image: url(img/project2.png);
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
+
+  .projectinfo2 {
+    display: flex;
+    flex-direction: column;
+    background-color: #1a2236;
+    width: 100%;
+    height: 70%;
+    justify-content: space-evenly;
+    padding-left: 2%;
+  }
```

Comment:
- [ ] the list of the languages doesn't fit the layout of the project card. I would recommend using `flex-wrap: wrap`. It defines whether the flex items are forced in a single line or can be flowed into multiple lines.

<img width="245" alt="image" src="https://user-images.githubusercontent.com/77190203/161175787-d2e62009-cc88-4bb8-9a44-9e2870ebc25a.png">